.dockerignore

```diff
@@ -1 +0,0 @@
-.gitignore
```

Comment:
What about this?
```
!**/*.go
!**/*.mod
!**/*.sum
```

Reply:
It means source files will not be included. But the docker script will actually build binary in the docker, so source files are needed.

Reply:
I think we need `Makefile`, too. We need to keep adding new source file types into the list. It makes no difference with adding new ignoring file types.